pages/maps/[id]/index.js

```diff
@@ -11,11 +11,17 @@ import {v4 as uuidv4} from "uuid";
 
 const MapInfoBox = styled.div`
 	display: flex;
+	flex-direction: column;
 	justify-content: space-between;
 	padding: 15px;
 	background: var(--color-bg-box-info);
 `;
 
+const MapInfoBoxText = styled.div`
```

Comment:
The name ending with `...Text` don't suggest it's a `div` element (because `div`s are only container elements for other HTML elements.

I think `display:block` is the default for both `h2` and `p`, so you don't need it in this case. 

What about making the `MapInfoBox` a `section` or an `article` and renaming this `...Text` thing?